index.xml

```diff
@@ -205,8 +206,11 @@ structured suffix processing rules, it cannot presume that a document that is
 valid per the decoding rules associated with the structured suffix
 will be valid for a recognized subset of the structured suffix. For example,
 presuming a media type of "application/foo+bar+baz", a toolchain cannot presume
-that a valid "+baz" document will also be a valid "+bar+baz" document or a
-valid "application/foo+bar+baz" document.
+that a valid "+baz" document will also be a valid "+bar+baz" document nor a
+valid "application/foo+bar+baz" document. On the other hand, presuming a media
+type of "application/foo+bar+baz", a toolchain CAN presume that a valid 
```

Comment:
```suggestion
type of "application/foo+bar+baz", a toolchain can presume that a valid 
```

I'm concerned that someone will think the "CAN" is normative, even though it isn't.

Reply:
I hear you, but what else do RFCs use for emphasis? `*can*`? `_can_`?

Reply:
This: https://ietf-tools.github.io/xml2rfc/#name-em

I commited the change using \<em>